tools/hangar/__snapshots__/test_corpus/sdk_tests/container/mount.test.w_test_sim.md

```diff
@@ -2,7 +2,6 @@
 
 ## stdout.log
 ```log
-[INFO] my test | Pulling postgres:15...
```

Comment:
(running the tests locally changed this snapshot) I'm not sure these belong at the INFO log level

Reply:
ok - I changed some of them to be verbose. Open to alternative ideas (another option could be to set the log level to `process.env.CI ? LogLevel.VERBOSE : LogLevel.INFO` or imagine some other way to scrub these from snapshots)